specification/iothub/resource-manager/Microsoft.Devices/2017-07-01/iothub.json

```diff
@@ -2138,6 +2441,28 @@
       "description": "The version of the API.",
       "required": true,
       "type": "string"
+    },
+	"resourceGroupName": {
```

Comment:
https://github.com/Azure/azure-openapi-validator/issues/84

Reply:
@amarzavery I've added "x-ms-parameter-location": "method" to the objects that describe parameters that are only used in methods, however I'm losing the "credentials" parameter `IotHubClientConfiguration(credentials, subscription_id, base_url)` has now become `IotHubClientConfiguration(subscription_id, base_url)` and this seems to be causing several issues. Any thoughts on how to fix this?

Reply:
Ah seems to be a autorest option that might be new